guide_drafts/backend_installation.md

```diff
@@ -0,0 +1,74 @@
+# Installing Backend Client Libraries
+
+Diesel supports SQLite, PostgreSQL, and MySQL as backend databases. By default, `diesel_cli` requires the client library of all three backends to be installed. If one is missing, then `cargo install diesel_cli` you will throw an error like:
+
+```
+note: ld: library not found for -lmysqlclient
+clang: error: linker command failed with exit code 1 (use -v to see invocation)
+```
```

Comment:
I always add empty lines between code blocks and paragraphs as it improves readability a bit (easier to see segments of text) but Markdown doesn't require it